### DISCUSSION TITLE
Announcement: Rust client library!

### DISCUSSION THREAD

**Discussion by phayes** [gotenberg/gotenberg#1071] [Q&A]
Hi Everyone,

Thanks for the great project. I'm currently working on a rust client library for gotenberg. You can find it here:

1. https://github.com/phayes/gotenberg_pdf
2. https://crates.io/crates/gotenberg_pdf
3. https://docs.rs/gotenberg_pdf/latest/gotenberg_pdf/

Feedback and comments appreciated! 

After I get some feedback and have used it significantly for myself, I'll release a v1 version.

**Comment by gulien** [comment]
Hey @phayes! Thanks for your work! Can you open a PR in https://github.com/gotenberg/awesome-gotenberg?
